src/controllers/write/bugs.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const bugController = module.exports;
```

Comment:
Like above, `bugsController` and `bugsCreate` to be consistent

Reply:
Fixed